<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoCCFWidgetUI</class>
 <widget class="QWidget" name="MoCCFWidgetUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MoCGF-GUI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="moCGFMainView">
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generatorTab">
      <attribute name="title">
       <string>Generators</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QListWidget" name="generatorList">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QToolButton" name="generatorListReloadButton">
               <property name="toolTip">
                <string notr="true" comment="Refresh list of generators"/>
               </property>
               <property name="text">
                <string>R</string>
               </property>
               <property name="icon">
                <iconset resource="Icons.qrc">
                 <normaloff>:/Icons/Reload-icon.png</normaloff>:/Icons/Reload-icon.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="generatorExplorerButton">
               <property name="toolTip">
                <string notr="true" comment="Open generator folder in file explorer"/>
               </property>
               <property name="text">
                <string>E</string>
               </property>
               <property name="icon">
                <iconset resource="Icons.qrc">
                 <normaloff>:/Icons/Folder-icon.png</normaloff>:/Icons/Folder-icon.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QTextBrowser" name="generatorView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>0</height>
           </size>
          </property>
          <property name="openExternalLinks">
           <bool>false</bool>
          </property>
          <property name="openLinks">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Data URIs:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="uriInput"/>
           </item>
           <item row="0" column="2">
            <widget class="QToolButton" name="uriLoadButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Output:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="outputInput"/>
           </item>
           <item row="1" column="2">
            <widget class="QToolButton" name="outputLoadButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="generateButton">
             <property name="toolTip">
              <string notr="true" comment="Generate output with this generator"/>
             </property>
             <property name="text">
              <string>Generate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="openOutputButton">
             <property name="toolTip">
              <string notr="true" comment="Open output in external program"/>
             </property>
             <property name="text">
              <string>Open output</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="apiTab">
      <attribute name="title">
       <string>Data APIs</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QListWidget" name="apiList"/>
         <widget class="QTextBrowser" name="apiView">
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>0</height>
           </size>
          </property>
          <property name="openExternalLinks">
           <bool>false</bool>
          </property>
          <property name="openLinks">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
